### ISSUE TITLE
Type with PostGIS `geography` can't be saved using console editor

### ISSUE THREAD

**Issue by michaelhayman** [hasura/graphql-engine#2665]
I'm 90% sure that this is the issue; on other sites on records without the geography type everything works.

I get an error when editing such records.

**Comment by marionschleifer** [comment]
@michaelhayman could you please share a screenshot showing what exactly you wanted to edit and ideally also the logs? Thank you 🙂

**Comment by ManUtopiK** [comment]
I think I got the same issue with `PosGIS geoghaphy`. When I edit a row in Hasura console, the geography value is displayed as `[object Object]`, not an object like:
```json
{
    "type": "Point",
    "crs": {
        "type": "name",
        "properties": {
            "name": "urn:ogc:def:crs:EPSG::4326"
        }
    },
    "coordinates": [
        -55.162353515625,
        5.82301611438301
    ]
}
```
So, when I want to save this edited row, I got this error:

    Edit failed!
    unexpected : unexpected character (at offset 1)

And I can't save any modifications...  
This is annoying, I have to change data from graphiql 😯 

<details><summary>Full error details</summary>

```js
{
    "statement": "WITH \"public_photo__mutation_result_alias\" AS (UPDATE \"public\".\"photo\" SET \"status\" = $1,\"uploadedAt\" = $2,\"wpID\" = $3,\"path\" = $4,\"createdAt\" = $5,\"location\" = ST_GeomFromGeoJSON($6 ),\"size\" = $7,\"medium\" = $8,\"country\" = $9,\"data\" = $10,\"format\" = $11,\"alert\" = $12,\"content\" = $13,\"userId\" = $14,\"base64\" = $15,\"id\" = $16,\"price\" = $17,\"updatedAt\" = $18,\"title\" = $19,\"stock\" = $20,\"agence\" = $21,\"file\" = $22  WHERE (('true') AND ((((((\"public\".\"photo\".\"id\") = ($23)) OR (((\"public\".\"photo\".\"id\") IS NULL) AND (($23) IS NULL))) AND ('true')) AND ('true')) AND ('true'))) RETURNING * ) SELECT  json_build_object('affected_rows', (SELECT  COUNT(*)  FROM \"public_photo__mutation_result_alias\"      ) )        ",
    "prepared": true,
    "error": {
        "exec_status": "FatalError",
        "hint": null,
        "message": "unexpected character (at offset 1)",
        "status_code": "XX000",
        "description": null
    },
    "arguments": [
        "(Oid 25,Just (\"publish\",Binary))",
        "(Oid 1184,Just (\"\\NUL\\SOH\\249\\245\\217\\DC4J@\",Binary))",
        "(Oid 23,Just (\"\\NUL\\NUL\\NAK\\206\",Binary))",
        "(Oid 25,Just (\"uploads/2017/08/DSC_7728._PRINT.jpg.zip\",Binary))",
        "(Oid 1184,Just (\"\\NUL\\SOHQ\\DC1SDu\\128\",Binary))",
        "(Oid 0,Just (\"[object Object]\",Text))",
        "(Oid 23,Just (\"\\NUL3\\135\\ESC\",Binary))",
        "(Oid 25,Just (\"reflex\",Binary))",
        "(Oid 25,Just (\"Suriname\",Binary))",
        "(Oid 3802,Just (\"\\SOH{\\\"height\\\":2955,\\\"osm_id\\\":26185592,\\\"width\\\":4461,\\\"address\\\":{\\\"state\\\":\\\"Paramaribo\\\",\\\"county\\\":\\\"Distrikt Paramaribo\\\",\\\"country_code\\\":\\\"sr\\\",\\\"country\\\":\\\"Suriname\\\",\\\"road\\\":\\\"Timmermanstraat\\\",\\\"city\\\":\\\"Paramaribo\\\",\\\"postcode\\\":\\\"3001\\\"},\\\"exif\\\":{\\\"FocalLengthIn35mmFilm\\\":43,\\\"ShutterSpeedValue\\\":8.321928,\\\"Copyright\\\":\\\"Paramaribo SURINAM\\\",\\\"DateTimeOriginal\\\":\\\"2011:09:29 13:10:30\\\",\\\"ApertureValue\\\":6.33985,\\\"FocalLength\\\":29,\\\"ExposureTime\\\":3.125e-3,\\\"Software\\\":\\\"Adobe Photoshop CC 2015 (Macintosh)\\\",\\\"Model\\\":\\\"NIKON D7000\\\",\\\"Make\\\":\\\"NIKON CORPORATION\\\",\\\"Flash\\\":\\\"Flash did not fire, compulsory flash mode\\\",\\\"ExposureMode\\\":\\\"Auto exposure\\\",\\\"ISOSpeedRatings\\\":200,\\\"Orientation\\\":\\\"top-left\\\",\\\"Artist\\\":\\\"Pieter Van Maele  +597 815 53 30\\\"}}\",Binary))",
        "(Oid 25,Nothing)",
        "(Oid 25,Just (\"\",Binary))",
        "(Oid 25,Just (\"A policeman salutes his superior during a parade on Independence Square in Paramaribo, Suriname.29/09/11\",Binary))",
        "(Oid 0,Just (\"fa8407b4-ec23-4a84-bb7a-89b1170134e9\",Text))",
        "(Oid 25,Just (\"data:image/jpeg;base64,/9j/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCAAeAC0DASIAAhEBAxEB/8QAGgAAAgMBAQAAAAAAAAAAAAAABQYCBAcBCP/EAC4QAAEEAQQABAYABwAAAAAAAAECAwQRBQAGEiEUMUFRBxMVIjJhIyQzYnGBof/EABcBAAMBAAAAAAAAAAAAAAAAAAECBAP/xAAfEQACAgIBBQAAAAAAAAAAAAABAgARAzFhBBIhQaH/2gAMAwEAAhEDEQA/ACuX+JmQTj7Q6gNyObaHozRH3DzTZFpI8rqjXRPnpZjbxy0F2PKflLLv9X5Tb5ctVXxUAej6EH31KBun6xvN2HjWkxvEsPK5KbT/ABFBskn7SKtQPl61o18Ps0vMN5FtpmDI8MgcluRyVKWQeKSSAPMd/wDNROvYxZiTXM0GO9GMub3S43AMmVkFMvpQoJaj8wC4KPAgCwe6s9exOgOM3BJzDjKZAcxENabTIkkpac77B6vl70PWzrju6cI/iUSMpLZEx9orW2pwkIeT0DxP41QoHzA1XwGTxGQnJU1DiLcQyVS0obpt9KTTZ5We1KKuuiReqGB3ZgCgHiB9wYLNJSqVElwSlbqvlOIlobChaglSSpQBFDu/fSGZEuEwwXfCuLdTyJXJQBf6o9fsHWloZgP5+TG+lwpy/p/jVNuRftQQPsBT5WaCa9eN+upsyIW9cnkHMc06hiKpAWeKWip0ppRJR0fwHoK1C3SYVBoVVfYQlwN8PNpzcVvSVPcZdYMZ11qMVMGngpRtdFBBHE1fXemL4a4qft7J5lLjcplmTMJTUYqC0DlRBKSAO/1rRfEG61z59k9atZr3CBWp5431tGTidzGOoO+AmSm/Crv8+6Kf8jlorlNoy8Hu84/CF8yZQQmGegW27taln+0dWR3ZrWsbnxsTKwv5xhtxce3WVOJ5fLWPJQH+tCsTkltsMOu46GmY02WAphRSA3zJAsgns0TZ8/1WtBkNRCvmUNiOmXubczpRKYlMvDHBDShwbabBCeRV93Zs6l8O9uztp4l5hUfICW+8px1caQ1wULIR0T0a9P3qrtfd8PLb9yTDGKMWe61b8gO8vmBFUKoe570+F9fvpH3R9xln/9k=\",Binary))",
        "(Oid 0,Just (\"0b77509d-e4b8-49ca-be19-0c0d1290d27e\",Text))",
        "(Oid 21,Just (\"\\NUL\\SOH\",Binary))",
        "(Oid 1184,Just (\"\\NUL\\STX2\\157\\177\\152\\159\\b\",Binary))",
        "(Oid 25,Just (\"Salute\",Binary))",
        "(Oid 16,Just (\"\\SOH\",Binary))",
        "(Oid 16,Just (\"\\NUL\",Binary))",
        "(Oid 25,Just (\"DSC_7728._PRINT.jpg\",Binary))",
        "(Oid 0,Just (\"0b77509d-e4b8-49ca-be19-0c0d1290d27e\",Text))"
    ]
}
```
**(Oid 0,Just (\"[object Object]\",Text))" is geography point**
</details>

This issue https://github.com/hasura/graphql-engine/issues/2651 seem to fix geography in data table, but they forgot to fix in row editor...

**Comment by rikinsk13** [comment]
Closed via #2843
